packages/jest-haste-map/src/types.js

```diff
@@ -23,3 +24,11 @@ export type WorkerMetadata = {
   dependencies: ?Array<string>,
 };
 export type WorkerCallback = (error: ?Error, metaData: ?WorkerMetadata) => void;
+
+export type CrawlerOptions = {
```

Comment:
That's true, yes! I realized that most of the objects I'm dealing with I want to have *exactly* the type I'm asking for so I think this should be the default. I think in Jest we have more cases with exact objects than without.